skipruntime-ts/tests/examples/departures.sh

```diff
@@ -2,12 +2,24 @@
 set -e
 cd ../../examples
 
-if [ "$#" -ne 2 ]; then
-    echo "Usage: $0 <stdout target> <stderr target>"
+cleanup() {
+    jobs -p | xargs kill
+}
+
+trap cleanup EXIT
+
+if [ "$#" -lt 2 ]; then
+    echo "Usage: $0 <stdout target> <stderr target> <platform>"
+fi
+
+if [ "$3" = "native" ]; then
+    echo "Run departures with native platform"
+    LD_LIBRARY_PATH=$(realpath ../../build/skipruntime) node dist/departures.js >/dev/null &
```

Comment:
This line (and similar throughout, excluding `database.sh`) need `-p native` flags, right?  If I'm reading right, then they're all still running under Wasm.